Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -529,13 +529,26 @@ def extract_namespace(response_items: List[Dict[str, Any]]):
                 break
 
 
-def remove_additional_resource_fields(input_dict, keys):
-    if isinstance(input_dict, dict):
-        return {k: remove_additional_resource_fields(v, keys) for k, v in input_dict.items() if k not in keys}
-    elif isinstance(input_dict, list):
-        return [remove_additional_resource_fields(element, keys) for element in input_dict]
-    else:
-        return input_dict
+def remove_additional_resource_fields(input_dict):
+    items = demisto.get(input_dict, 'data.items')
+    if items:
+        for current_item in list(items):
+            data = current_item['data']
```

Comment:
use get instead of []

Reply:
[Done](https://github.com/demisto/content/pull/32714/commits/74d3174195848c67a55b1a71ef6ca1ddbd9a7713).